src/check/parse/Parser.zig

```diff
@@ -18,6 +18,7 @@ tok_buf: TokenizedBuffer,
 store: IR.NodeStore,
 scratch_nodes: std.ArrayList(IR.Node.Idx),
 diagnostics: std.ArrayList(IR.Diagnostic),
+// log_file: std.fs.File,
```

Comment:
Did you intend to leave this in?

Reply:
I thought about leaving it in, but I want to wait for us to have a holistic logging solution for the entire compiler.

Reply:
Removed